src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -220,7 +239,72 @@ public String getRemote() {
         }
     }
 
-    protected abstract void doRetrieve(SCMHeadObserver observer, TaskListener listener, GHRepository repo) throws IOException, InterruptedException;
+    protected void doRetrieve(SCMHeadObserver observer, TaskListener listener, GHRepository repo) throws IOException, InterruptedException {
```

Comment:
@jglick I agree about the modifier. I left `retrieve()` and `doRetrieve()` in order to avoid a long method. IMHO, it is clear if splitted.